core/core.md

```diff
@@ -62,14 +62,16 @@ Thus the Fora protocol must assume that malicious actors will never reach the th
+While it is important that new members are able to join in order to keep the community dynamic, malicious admittors can use their power to flood the community with new members that are controlled by the admittor and thus take over the governance of the community. Thus, to prevent this we enforce a delay period (parameterized by governance) on the voting power of the new members. Thus, while newly admitted members can immediately start interacting in the community; the delay period must pass first before their voting power becomes active. The delay period **must** be greater than the voting period necessary to change the admittor. The true voting power an admittor can grant a single user is also **proportional** to the admittors own weight. For example, if a single admittor holds 1/3 the voting power, the sum of all voting power this admittor grants equals 1/3. An admittors voting power can only be changed through a governance proposal.
 
-This gives the community enough time to detect the attack and replace the Admittor before the Admittor takes over community governance, while also allowing new members to immediately participate in the community and giving them voting power later on.
+This gives the community enough time to detect the attack and replace the Admittor before the Admittor takes over community governance, while also allowing new members to immediately participate in the community and giving them voting power later on. Admittors who try to admit malicious members over a longer period of time are also constrained by their own voting power obtained through a governace proposal.
 
-If the Admittor gets replaced for malicious activity, then any member that is still in the delay period will have their voting power remain 0. The new Admittor must **explicitly** choose to give back the new members their voting power, at which point the delay period gets reset and must pass again from that point. However, these members can still interact with the community unless the admittor or enforcer **explicitly** kicks them out of the community.
+If the Admittor gets replaced for malicious activity, then any new member that is still in the delay period will have their voting power remain 0. The voting power given to users from this admittor will also be removed. The new Admittor must **explicitly** choose to give back the new members their voting power, at which point the delay period gets reset and must pass again from that point. However, these members can still interact with the community unless the admittor or enforcer **explicitly** kicks them out of the community.
```

Comment:
Hmmm... I'm not sure this should happen because it places an explicit incentive for members to protect the admittors that gave them voting power. Imagine if you are a malicious admittor, and half of your members are good members and half of your members are bad members. Voting to replace you, will cause all of your member's voting power to go to 0. This might disincentivize honest members from voting against you, since it is a vote to disenfranchise themselves.

We should keep the voting power of the established members the same, since the admittor DID controller just gets replaced even though the admittor entity has the same voting power and member list. The new admittor can then choose to explicitly set some of his members voting powers to zero, at which point the delay period kicks in.

We don't want to create perverse incentives in the protocol. Ideally, only dishonest members vote for dishonest authorities. If there's dishonest members + self-interested honest members voting for dishonest authorities, we may not be able to replace them easily.

Reply:
I see. Can admittors choose to reduce one's voting powers? I was thinking of the case where users can never have there voting power reduced. Maybe this is okay though since enforcers should take action on "bad" users